GPU Error with too large numbers

This One-line Code in GPU :`db 11111111111111111111111111111111111111111111111;` creates the error
```
[ERROR] addons/wire/lua/entities/gmod_wire_gpu/cl_gpuvm.lua:67: attempt to concatenate local 'interruptParameter' (a nil value)
  1. Interrupt - addons/wire/lua/entities/gmod_wire_gpu/cl_gpuvm.lua:67
   2. unknown - ZVM:[0]:8
    3. Step - addons/wire/lua/wire/zvm/zvm_core.lua:619
     4. Run - addons/wire/lua/entities/gmod_wire_gpu/cl_init.lua:159
      5. RenderGPU - addons/wire/lua/entities/gmod_wire_gpu/cl_init.lua:246
       6. unknown - addons/wire/lua/entities/gmod_wire_gpu/cl_init.lua:351
```
The GPU Code has generally some problems handling too large numbers.

However the code `db 1111111111111111111111;` doesn't create any error.

--- a/lua/entities/gmod_wire_gpu/cl_gpuvm.lua
+++ b/lua/entities/gmod_wire_gpu/cl_gpuvm.lua
@@ -43,7 +43,7 @@ function ENT:OverrideVM()
     if self.ASYNC == 1 then
       if self.EntryPoint5 > 0 then
         self.IP = self.EntryPoint5
-        self.LADD = interruptParameter
+        self.LADD = interruptParameter or self.XEIP
         self.LINT = interruptNo
       else
         -- Shutdown asynchronous thread
@@ -52,7 +52,7 @@ function ENT:OverrideVM()
     else
       if self.EntryPoint3 > 0 then
         self.IP = self.EntryPoint3
-        self.LADD = interruptParameter
+        self.LADD = interruptParameter or self.XEIP
         self.LINT = interruptNo
       else
         if (interruptNo == 2) and (self.XEIP == 0) then self.INTR = 1 return end
@@ -64,7 +64,7 @@ function ENT:OverrideVM()
 		  
           draw.DrawText("Error in the instruction stream","WireGPU_ErrorFont",48,16,Color(255,255,255,255))
           draw.DrawText((self.ErrorText[interruptNo] or "Unknown error").." (#"..interruptNo..")","WireGPU_ErrorFont",16,16+32*2,Color(255,255,255,255))
-          draw.DrawText("Parameter: "..interruptParameter,"WireGPU_ErrorFont",16,16+32*3,Color(255,255,255,255))
+          draw.DrawText("Parameter: "..(interruptParameter or 0),"WireGPU_ErrorFont",16,16+32*3,Color(255,255,255,255))
           draw.DrawText("Address: "..self.XEIP,"WireGPU_ErrorFont",16,16+32*4,Color(255,255,255,255))
 
           local errorPosition = CPULib.Debugger.PositionByPointer[self.XEIP]

